pori_python/ipr/content.spec.json

```diff
@@ -99,12 +99,11 @@
             "items": {
                 "properties": {
                     "chromosomeBand": {
-                        "example": "X:p12.2",
-                        "pattern": "^(\\S+:\\S+?)?$",
+                        "example": "Xp12.2",
                         "type": "string"
                     },
                     "cna": {
-                        "description": "The copy number alteration (CNA) ratio",
+                        "description": "Copy Number, Absolute (cna)",
```

Comment:
With this description, it makes sense to use to use this field for the copy number tool output, unaltered.

So Purple can upload non-integer copy numbers and the older CNASEQ can still have its raw copy number alteration uploaded.

Reply:
awesome. I suspected the new pipeline had something to do with it